routes/scheduleStack.js

```diff
@@ -0,0 +1,23 @@
+import { createStackNavigator } from "react-navigation-stack";
+import CourseSchedule from "../screens/Menu/CourseSchedule";
+import CourseScheduleDetails from "../screens/Menu/CourseScheduleDetails";
+
+const screens = {
+
+    CourseSchedule: {
+        screen: CourseSchedule,
+        navigationOptions: {
+            headerShown: false,
+        }
+    },
+    CourseScheduleDetails: {
+        screen: CourseScheduleDetails,
+        navigationOptions: {
+            headerShown: false,
+        }
+    }
+};
+
+const CourseScheduleStack = createStackNavigator(screens);
+
+export default CourseScheduleStack; 
```

Comment:
<img width="869" alt="Screen Shot 2020-04-01 at 4 22 21 PM" src="https://user-images.githubusercontent.com/35850933/78182869-f83ba000-7434-11ea-944e-f6791c2a0cda.png">

Reply:
Does not make sense for this one

Reply:
Ok can change it!